build.sbt

```diff
@@ -152,7 +152,7 @@ lazy val cats = (project in file("cats"))
       "AWS_SECRET_KEY" -> "credentials"
     ),
     dynamoDBLocalDownloadDir := file(".cats-effect-dynamodb-local"),
-    dynamoDBLocalPort := 8042,
+    dynamoDBLocalPort := 8043,
```

Comment:
I don't understand how this can work when `LocalDynamoDB.client()` connects to port 8042 😮 This must be a sign there's some resource leak.